components/Form/index.js

```diff
@@ -0,0 +1,34 @@
+const T = require('prop-types');
+const { KeyboardAwareScrollView } = require('react-native-keyboard-aware-scroll-view');
+const { default: Styled } = require('styled-components/native');
+
+const internals = {};
+
+module.exports = function Form({ isDirty, onSubmit, submitLabel, children, disabled, ...props }) {
+
+    const { Container } = internals;
+
+    return (
+        <Container {...props} keyboardShouldPersistTaps='handled'>
+            {children}
+        </Container>
+    );
+};
+
+module.exports.propTypes = {
+    children: T.any.isRequired,
+    onSubmit: T.func,
+    isDirty: T.bool,
+    disabled: T.bool,
+    submitLabel: T.string
+};
+
+module.exports.defaultProps = {
+    isDirty: false,
+    disabled: false,
+    submitLabel: 'Save Changes'
+};
+
+internals.Container = Styled.ScrollView`
```

Comment:
Asking just because there were no styles for this one. Can we directly use `ScrollView` or is there some unexpected behavior with that approach?

Reply:
I've updated this, but we may also just wish to cut it; I'm not sure if we've been using it much!

Reply:
ohh I'm not sure what's the best thing to do for this one 😅